client/my-sites/plugins/plugin-automated-transfer/index.jsx

```diff
@@ -80,7 +80,7 @@ class PluginAutomatedTransfer extends Component {
 			return translate( "Don't leave quite yet! Just a bit longer." );
 		}
 		if ( transferComplete ) {
-			return translate( 'Successfully installed %(plugin)s!', { args: { plugin: plugin.name } } );
+			return translate( 'Activating %(plugin)s...', { args: { plugin: plugin.name } } );
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Activating one plugin', 'Activating %(numberOfPlugins)d plugins' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).